packages/backend-common/src/database/DatabaseManager.ts

```diff
@@ -71,11 +78,15 @@ export class DatabaseManager {
    */
   forPlugin(pluginId: string): PluginDatabaseManager {
     const _this = this;
+    const defaultMigrationOptions = {
+      apply: true,
```

Comment:
Best default just apply in isolation tbh, rather than the entire migrations object. That way as we add more things the apply would be defaulting like we want it to